deploy/role.yaml

```diff
@@ -31,7 +31,6 @@ rules:
 - apiGroups:
   - apps
   resources:
-  - replicasets
```

Comment:
replicasets is required. You will get following error during operator startup without this permission  

```
{"level":"info","ts":1605865859.5327392,"logger":"cmd","msg":"Could not create metrics Service","error":"failed to initialize service object for metrics: replicasets.apps \"infinispan-operator-85c5568b5c\" is forbidden: User \"system:serviceaccount:infinispan-operator:infinispan-operator\" cannot get resource \"replicasets\" in API group \"apps\" in the namespace \"infinispan-operator\""}
```

Reply:
ouch, I though it was a relic from the first version. thanks @dmvolod

Reply:
No, @rigazilla I validated all possible rights during role cleanup